### ISSUE TITLE
marathon.app.list doesn't seem to work

### ISSUE THREAD

**Issue by jamsea** [silas/node-mesos#2]
Let me know if I'm crazy and I'm just using this wrong. I'm using version 0.8.1 of marathon.

I tried testing this package with this code:
```
var mesos = require('mesos'); 
var marathon = mesos.Marathon({ host: process.env.MARATHON_HOST }); 
marathon.app.list(function(stuff) { console.log(stuff) });
```

However what I'm receiving is:
```
{ [Error: marathon: app.list: not found] message: 'marathon: app.list: not found' }
```

Any ideas? I'm fine with investigating a fix for a package if there's a problem with the package. Just want to know that I'm not using it wrong :)

**Comment by silas** [comment]
No, it's definitely pretty broken right now. There was a pretty big upgrade to the marathon api soon after I built this and I haven't gotten around to updating it yet.
